CMakeLists.txt

```diff
@@ -185,10 +185,10 @@ if (NOT EMBEDPLUGINS AND NOT BUILD_OIIOUTIL_ONLY)
     endforeach ()
 endif ()
 
-if (USE_PYTHON AND boost_PYTHON_FOUND AND NOT BUILD_OIIOUTIL_ONLY)
+if (USE_PYTHON AND Boost_PYTHON_FOUND AND NOT BUILD_OIIOUTIL_ONLY)
```

Comment:
What's up with the capitalization change? Did we have this wrong all along? Why did it work before on the other platforms? Is it possible that this is something that changed at some point and might be different for other Boost versions?

Reply:
I believe since you're using `find_package(Boost ...)`, the general pattern is that this module should set `Boost_FOUND` and similar. That is also what the CMake source-code says: https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/FindBoost.cmake#L27

Reply:
Agreed.